src/components/coursesSection/CoursesSection.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Courses from "./courses/Courses";
+import coursesSectionStyle from "./CoursesSection.module.css";
+
+function coursesSection() {
```

Comment:
- the component name should start with an uppercase letter.
- title, description, and course list should be passed to this component as props.

Reply:
* I assumed that the component name is the name of the function, if it is not the case, please let me know
* I placed the title, description, and courses list in the index.js file and passed them as props to their required position

Reply:
according to title, description, and courses I mean you pass them from app.js to this component not from index.js but it is ok for now. please fix it in the next task.